### PR TITLE
Initial commit for the JS library for epic logger

### PR DESCRIPTION
**Pull request by codelion**
Added the following 
- core js library that will be installed by users
- contact form that will allow submitting the information
- test for the core js library (currently posts the date to a firebase db).

### CODE REVIEW

File: test/contact.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                + currentdate.getSeconds();
+        myFirebaseRef.push({
+          'id' : id,
+          'email' : formObj.InputEmail.value,
+          'notes' : formObj.InputMessage.value,
+          'timestamp': datetime
+        }, onSubmitComplete);   
+        contactBtn.disabled = true;
+        return false;
+    }
```
Comment (Chocksy) [inline test/contact.html]: I think we should move this part of code and update the loader script. This code should be integrated so we don't load have js in 2 places even though i understand that it's for submitting the data and that's why you did it but what i had in mind was that we should have even the html and css part in the script so we don't load an iframe any more. 

So the loading will be super fast and we don't have to host this page anywhere on any server to load it. Seems fair?

**Comment by codelion** [comment]
Moved all the code into loader script and use the pico modal for loading it instead of iframe.

**Comment by Chocksy** [comment]
@codelion let's remove the jquery file and bootstrapjs file as we are not using them i hope. And i would like in the future to remove the css file too. Once you clean that up let me know. I'll have another look and merge it. :D Thanks for the work.

**Comment by codelion** [comment]
@Chocksy done :)

**Comment by Chocksy** [comment]
@codelion i'm going to merge this now and i'm going to make another bounty to clean things up and make a minified version and also make the code more specific to our needs as now we have some functions we might not need.

**Pull request merged**